packages/lisk-transactions/src/5_dapp_transaction.ts

```diff
@@ -260,6 +261,11 @@ export class DappTransaction extends BaseTransaction {
 			);
 		}
 		const validLinkSuffix = ['.zip'];
+
+		if (errors.length > 0) {
```

Comment:
why return here?

Reply:
If asset does not pass validation we should not check fields like `this.asset.dapp.link` below, otherwise it will throw an error.